SecurityException

```
FATAL EXCEPTION: main
Process: org.ole.planet.myplanet, PID: 7470
java.lang.SecurityException: Need to declare android.permission.REQUEST_INSTALL_PACKAGES to call this api
	at android.os.Parcel.createExceptionOrNull(Parcel.java:3383)
	at android.os.Parcel.createException(Parcel.java:3367)
	at android.os.Parcel.readException(Parcel.java:3350)
	at android.os.Parcel.readException(Parcel.java:3292)
	at android.content.pm.IPackageManager$Stub$Proxy.canRequestPackageInstalls(IPackageManager.java:8017)
	at android.app.ApplicationPackageManager.canRequestPackageInstalls(ApplicationPackageManager.java:3745)
	at org.ole.planet.myplanet.base.BasePermissionActivity$Companion.hasInstallPermission(BasePermissionActivity.kt:503)
	at org.ole.planet.myplanet.base.BaseContainerFragment.onCreate(BaseContainerFragment.kt:62)
	at androidx.fragment.app.Fragment.performCreate(Fragment.java:3094)
	at androidx.fragment.app.FragmentStateManager.create(FragmentStateManager.java:504)
	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:268)
	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1943)
	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1839)
	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1782)
	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:3042)
	at androidx.fragment.app.FragmentManager.dispatchActivityCreated(FragmentManager.java:2952)
	at androidx.fragment.app.FragmentController.dispatchActivityCreated(FragmentController.java:263)
	at androidx.fragment.app.FragmentActivity.onStart(FragmentActivity.java:350)
	at androidx.appcompat.app.AppCompatActivity.onStart(AppCompatActivity.java:238)
	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1718)
	at android.app.Activity.performStart(Activity.java:9373)
	at android.app.ActivityThread.handleStartActivity(ActivityThread.java:4583)
	at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:214)
	at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:194)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleItem(TransactionExecutor.java:166)
	at android.app.servertransaction.TransactionExecutor.executeTransactionItems(TransactionExecutor.java:101)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:80)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:3024)
	at android.os.Handler.dispatchMessageImpl(Handler.java:142)
	at android.os.Handler.dispatchMessage(Handler.java:125)
	at android.os.Looper.loopOnce(Looper.java:292)
	at android.os.Looper.loop(Looper.java:393)
	at android.app.ActivityThread.main(ActivityThread.java:9486)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:575)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:939)
Caused by: android.os.RemoteException: Remote stack trace:
	at com.android.server.wm.Session$$ExternalSyntheticBUOutline1.m(R8$$SyntheticClass:0)
	at com.android.server.pm.ComputerEngine.canRequestPackageInstalls(ComputerEngine.java:3960)
	at com.android.server.pm.IPackageManagerBase.canRequestPackageInstalls(IPackageManagerBase.java:213)
	at android.content.pm.IPackageManager$Stub.onTransact(IPackageManager.java:4070)
	at com.android.server.pm.PackageManagerService$IPackageManagerImpl.onTransact(PackageManagerService.java:6965)
```

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -12,8 +12,8 @@ android {
         applicationId "org.ole.planet.myplanet"
         minSdk = 26
         targetSdk = 36
-        versionCode = 5333
-        versionName = "0.53.33"
+        versionCode = 5334
+        versionName = "0.53.34"
         ndkVersion = '26.3.11579264'
         vectorDrawables.useSupportLibrary = true
     }

--- a/app/src/main/java/org/ole/planet/myplanet/base/BasePermissionActivity.kt
+++ b/app/src/main/java/org/ole/planet/myplanet/base/BasePermissionActivity.kt
@@ -20,6 +20,7 @@ import androidx.lifecycle.lifecycleScope
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.launch
 import kotlinx.coroutines.withContext
+import org.ole.planet.myplanet.BuildConfig
 import org.ole.planet.myplanet.R
 import org.ole.planet.myplanet.utils.Utilities
 

--- a/app/src/main/java/org/ole/planet/myplanet/base/BasePermissionActivity.kt
+++ b/app/src/main/java/org/ole/planet/myplanet/base/BasePermissionActivity.kt
@@ -500,7 +500,7 @@ abstract class BasePermissionActivity : AppCompatActivity() {
 
         @JvmStatic
         fun hasInstallPermission(context: Context): Boolean {
-            return context.packageManager.canRequestPackageInstalls()
+            return !BuildConfig.LITE && context.packageManager.canRequestPackageInstalls()
         }
     }
 }

